keycloak/README.md

```diff
@@ -0,0 +1,30 @@
+# The script to deploy Keycloak on Greenstand Kubernetes cluster
+
+## Prerequisites
+
+- Node.js
+- kubeclt ? 
+
+### To install prerequisites 
+
+```bash
+# install ansible
+pip3 install ansible
+
+# install k8s plugin
+ansible-galaxy collection install community.kubernetes
+```
+
+## Usage
+
+```bash
+chmod +x deploy.sh
```

Comment:
Can't you check it in with this flag set?

Reply:
Yea